lhcb-YYYY-open-data-curation/scripts/strippingpages/parse_stripping_table.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
Cosmetics: we try to standardise on using the `black` code formatter.  If OK with you, we could add pass these scripts via black too...

Reply:
There are two approaches to this, one was more manual, the other one should work a bit more automatically but needs to be checked.